c/array-examples/copysome1_false-unreach-call.i

```diff
@@ -0,0 +1,20 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
```

Comment:
In my opinion, the preprocessed files don't need to be provided here. The .set files can have patterns for .c files.

Reply:
The original file contains a `#define`. So either the preprocessed file needs to be kept, or the `#define` removed.

Reply:
We would like to keep #define. And prepocessed file is already present.